packages/config/utils/utils.test.js

```diff
@@ -3,55 +3,55 @@ import {getContracts, anyHasPrivateKeys} from "./utils"
 describe("getContracts", () => {
```

Comment:
I think here (or somewhere else not certain about the code org) a test is missing that would focus only on a JSON that looks something like (without aliases):
```json
{
    networks: {
        emulator: "127.0.0.1:3569",
    },
    accounts: {
        default: {
          address: "f8d6e0586b0a20c7",
          key: "ba68d45a5acaa52f3cacf4ad3a64d9523e0ce0ae3addb1ee6805385b380b7646",
        }
    },
    contracts: {
        HelloWorld: "./cadence/contracts/HelloWorld.cdc"
    },
    deployments: {
        emulator: {
           "default": ["HelloWorld"]
        }    
    }
}
```

And asserts that contract named "HelloWorld" is resolved to address "f8d6e0586b0a20c7" through the deployment link.

Reply:
This is the old version of the flow.json, yeah?

Reply:
This is the current version, the v2 is still in progress, so no need to stress about it yet...

Reply:
Thank you. That makes me feel better.